platform/darwin/src/MGLAccountManager.m

```diff
@@ -1,26 +1,37 @@
 #import "MGLAccountManager_Private.h"
 #import "NSBundle+MGLAdditions.h"
+
 #if TARGET_OS_OSX
 #import "NSProcessInfo+MGLAdditions.h"
 #endif
 
 #if TARGET_OS_IPHONE || TARGET_OS_SIMULATOR
 #import "MGLMapboxEvents.h"
+#endif
 
-@interface MGLAccountManager ()
+@interface MGLSKUToken : NSObject
 
-@property (atomic) NSString *accessToken;
-@property (nonatomic) NSURL *apiBaseURL;
++ (NSString *)skuToken;
+
+@end
+
+@implementation MGLSKUToken
```

Comment:
This is a stub to enable building, while the `mapbox-accounts-ios` library integration happens.

Reply:
→ d88ac48cf97ac477df238d24544ce07660278d62